pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+- [ ] Designate (One or More) Github Team(s) with `Write` Access to the repository
+- [ ] Submit a Pull Request to add the Github Repository for the Website to [GSA](https://github.com/GSA/.allstar/blob/main/allstar.yaml) or [GSA-TTS](https://github.com/GSA-TTS/.allstar/blob/main/allstar.yaml) .allstar Github Configuration Scanner)
+- [ ] Create a [`SECURITY.md` file](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository)
+- [ ] Follow GSA's [Digital Lifecycle Program](https://insite.gsa.gov/employee-resources/communications/websites/strategy-policy-and-standards/digital-lifecycle-program?term=DLP) for the Website
+- [ ] Confirm your Website on [Touchpoint's Registry](https://touchpoints.app.cloud.gov/admin/websites) or Submit a new website request in [Touchpoints](https://touchpoints.app.cloud.gov/admin/websites/new)
+
+## Authority to Use (ATU) Review
+Submit an [Authority to Use (ATU) Request](https://github.com/GSA-TTS/tts-pages/issues/new?)
+
+- [ ] Resolve any Critical or High security findings from Security Scanners
+- [ ] Resolve any Web Scan findings provided by GSA Security Team
+
+Once the review is completed the Site Manager will be sent their ATU Approval package via Docusign. The Site Manager will be responsible for managing Security Findings over the lifecycle of the Website.
+
+## Maintaining Approved Sites 
+Sites hosted on TTS Pages are required to have their URLs scanned in accordance with CIO-IT Security-06-30: Managing Enterprise Cybersecurity Risk and GSA’s parameter for National Institute of Standards and Technology (NIST) Special Publication (SP) 800-53, control RA-5, Vulnerability Scanning. 
+
+>This is performed after the ATU request is submitted
+
+## 	Reassessment 
+A Site’s ATU will have to be reassessed and an ATU reissued if the Site is found `NOT` to be in conformity with the requirements within this guide.
+
+Conditions/events that may require a reassessment of the ATU include:
+- New third party integrations not on the approved list are added.
+- The data types or information presented on the site changes.
+- A significant security incident occurs.
+- There are deviations from the ATU maintenance requirements.
+
+>This Determination is made by the TTS Pages System Owner `tts-tech-operations@gsa.gov`. Generally, this is done if security findings are not being addressed promptly.
+
+Website Managers will be notified, the following steps are only in the event that the Website Manager is none responsive.
+
+## Failure to Maintain Site - Site Removal
+Sites that fail to maintain the ATU requirements will be issued a formal notice. The TTS Pages team may take steps to disable the site or remediate the vulnerabilities. ATU site owners who hit certain triggers of overdue POA&Ms and/or failure to maintain alignment to ATU requirements will be required to provide a **Corrective Action Plan (CAP)** addressing the plan to address the deficiencies. 
```

Comment:
Can we spell out POA&M here, its the first time we use this acronym in this document

Reply:
This is the first time the document describes a team for TTS Pages. It's not immediately clear to me (and I'm on the cloud.gov Pages team) whether that means us :D. Who or what is the team? is it an office? Can you introduce or identify it, provide a contact for it, or otherwise disambiguate it from the cloud.gov pages team?

Reply:
Updated to introduce TY!